js/parts/Series.js

```diff
@@ -982,6 +982,10 @@ null
      *         Bar
      * @sample {highcharts} highcharts/plotoptions/series-stacking-percent-area/
      *         Area
+     * @sample {highcharts} highcharts/plotoptions/series-stacking-waterfall-normal
+     *         Column
```

Comment:
Wrong text. Should be Waterfall with normal stacking

Reply:
The _name_ still needs to be changed. "Column" is not the correct name.

Reply:
Ok, sorry for the misunderstanding. Corrected.